core/types/subtyping.cc

```diff
@@ -1386,57 +1386,4 @@ bool Types::equivNoUntyped(const GlobalState &gs, const TypePtr &t1, const TypeP
     return isAsSpecificAs(gs, t1, t2) && isAsSpecificAs(gs, t2, t1);
 }
 
-bool ClassType::derivesFrom(const GlobalState &gs, ClassOrModuleRef klass) const {
```

Comment:
I can see an argument for leaving these in subtyping.cc, but I don't feel super strongly about it.

Reply:
All the other implementations are in different files, so it felt reasonable to move them.